force-app/main/default/lwc/attendanceRow/attendanceRow.js

```diff
@@ -0,0 +1,107 @@
+/*
+ *
+ *  * Copyright (c) 2020, salesforce.com, inc.
+ *  * All rights reserved.
+ *  * SPDX-License-Identifier: BSD-3-Clause
+ *  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ *
+ */
+
+import { LightningElement, api, track, wire } from "lwc";
+import { getChildObjectByName } from "c/util";
+import { getRecord, getRecordNotifyChange } from "lightning/uiRecordApi";
+
+import SERVICE_DELIVERY_OBJECT from "@salesforce/schema/ServiceDelivery__c";
+import ID_FIELD from "@salesforce/schema/ServiceDelivery__c.Id";
+import QUANTITY_FIELD from "@salesforce/schema/ServiceDelivery__c.Quantity__c";
+import ATTENDANCE_STATUS_FIELD from "@salesforce/schema/ServiceDelivery__c.AttendanceStatus__c";
+
+// TODO: create design parameters for default status and "present" statuses
+const PRESENT_STATUS = "Present";
+
+export default class AttendanceRow extends LightningElement {
+    @track localRecord;
+    @track localFieldSet;
+
+    @api unitOfMeasurement;
+    @api presentStatus = PRESENT_STATUS;
+    @api readOnly = false;
+    name;
+    _isEdited;
+    recordId;
```

Comment:
Probably not a big deal, but caught my eye that we're using recordId here, but not in the typical @api context.